src/org/zaproxy/zap/extension/quickstart/resources/Messages_he_IL.properties

```diff
@@ -1,53 +0,0 @@
-#X-Generator: crowdin.com
```

Comment:
Should all localized .properties be removed or was this an accidental inclusion/change?